src/components/text-input/index.md.njk

```diff
@@ -84,9 +84,11 @@ Use prefixes and suffixes to help users enter things like currencies and measure
 
 {{ example({group: "components", item: "text-input", example: "input-prefix-suffix", html: true, nunjucks: true, closed: true, size: "s"}) }}
 
-Prefixes and suffixes are visual aids and are not available to assistive technologies. Instead, word the label to provide the necessary context.
+Prefixes and suffixes are useful when you use symbols and abbreviations that are commonly known and understood. Symbols and abbreviations should be explained in the label or hint. For example, put “Cost, in pounds” in the label, and use the “£” symbol in the text input prefix.
```

Comment:
I worked a little with prefixes and suffixes and I don't think its just useful for symbols and abbreviations? Words can be useful suffixes and prefixes too

Reply:
Yep. This is tricky. The first example has "per item" as a suffix too. I think what I was trying to get across is not to overload the prefix/suffix with long text.

Could the addition of an or statement work?

>Prefixes and suffixes are useful when you use symbols and abbreviations that are commonly known and understood or short amounts of text.

Reply:
I see - how about

> Avoid long text for prefixes and suffixes - stick to short text or commonly known symbols and abbreviations (for example 'km')

Reply:
It would be good to mention in the guidance that long text should be avoided 👍  

We actually asked the Working Group for examples of long prefixes/suffixes, here are some examples we received:
- per education authority
- per cubic metre
- of actual costs
- once per agreement
- per 100 square metres

Most of these probably shouldn't be included in the input suffix but I'm not sure how we should best communicate that in the guidance. Should we set a recommended character limit? It would need to the sum of the characters in the prefix and the suffix.